includes/class-database-cleanup.php

```
@@ -285,16 +408,26 @@ public static function clean_orphan_postmeta() {
+			'trashed_posts'      => 'clean_trashed_posts',
+			'spam_comments'      => 'clean_spam_comments',
+			'trashed_comments'   => 'clean_trashed_comments',
+			'expired_transients' => 'clean_expired_transients',
+			'orphan_postmeta'    => 'clean_orphan_postmeta',
 		);
 
+		$results = array();
+		foreach ( $methods as $key => $method ) {
+			$res = self::$method();
+			if ( false === $res ) {
+				$results[ $key ] = new \WP_Error( 'db_cleanup_failed', sprintf( '%s failed', $method ) );
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '425', column '28'). (undefined)

(MissingImport)